packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -127,45 +129,54 @@ class VideoPlayerAPI {
         // Ready callback is issued for this player:
         readyCallback: function(playerId) {
           var kdp = document.getElementById(playerId);
-          kdp.kBind('Play', function() {
-            return playerId;
+
+          kdp.addJsListener('playerPaused', function() {
+            fireEvent({ playerState: 1, kdp, videoId });
+          });
+          kdp.addJsListener('playerPlayed', function() {
+            fireEvent({ playerState: 2, kdp, videoId });
+          });
+          kdp.addEventListener('playerPlayEnd', function() {
```

Comment:
What's the difference between `addJsListener` and `addEventListener`?

Reply:
whoops left that in there by accident. `addJsListener` is a method specific to kaltura so `addEventListener` wont work - removing...